packages/core/src/Stores/client-store.js

```diff
@@ -882,26 +881,6 @@ export default class ClientStore extends BaseStore {
         return this.residence === 'gb';
     }
 
```

Comment:
@matin-deriv Are we removing below country logics in client-store with this PR scope
```
  get is_brazil() {
        return this.clients_country === 'br';
    }

    get is_uk() {
        return this.residence === 'gb';
    }

Reply:
`Is_brazil` is still needed as it is pending the BE API to handle the logic - WIP
`is_uk` simply missed from the card, good catch @amina-deriv.